sphinx/users/comlinetools/conversion.rst

```diff
@@ -270,3 +270,23 @@ correctly. So in Windows, the above example would read::
     The default fill value is typically 0, but some readers may have different defaults.
 
     .. versionadded:: 6.13.0
+
+.. option:: -precompressed
+
+    Enables transfer of compressed bytes from input dataset directly to output without decompression.  
+    Most input and output formats do not support this option, currently this is implemented in the 
+    Aperio SVS input and DICOM output. Do not use -crop, -fill, or -autoscale, or pyramid generation options::
+
+      bfconvert -precompressed /path/to/input /path/to/output
+
+    .. versionadded:: 7.1.0
+
+.. option:: -extra-metadata METADATA FILE
+
+    Enables the writing of additional supplemental metadata from an external file during the conversion process .
+    Most output formats do not support this option, currently this is implemented for outputting the DICOM format. 
+    For DICOM the additional metadata file can be in the format of a .dcdump or .json file ::
+
+      bfconvert -extra-metadata /path/to/extra-metadata /path/to/input /path/to/output
+
+    .. versionadded:: 7.1.0
```

Comment:
Worth linking to the original PRs for both of these, since the PR comments show working examples?

Reply:
I've copied one of the examples from the original PR and added it as a code sample for the extra-metadata option.
I don't if you have a good suitable sample for precompressed? Linking directly to the full PR review comments just seems a bit messy.

Reply:
Maybe just `bfconvert -noflat -precompressed -compression JPEG CMU-1.svs CMU-1.dcm` (from https://github.com/ome/bioformats/pull/3992#issuecomment-1830846748 and https://github.com/ome/bioformats/pull/3992#pullrequestreview-1754852581)?